synapse/models/crypto.py

```diff
@@ -321,20 +321,24 @@ def getModelDefs(self):
 
                 ('crypto:smart:effect:proxytoken', {}, (
                     ('owner', ('crypto:currency:address', {}), {
-                        'doc': 'The address granting proxy authority to manipulate a specific non-fungible token.'}),
+                        'doc': 'The address granting/denying proxy authority to manipulate non-fungible tokens.'}),
                     ('proxy', ('crypto:currency:address', {}), {
-                        'doc': 'The address granted proxy authority to manipulate a specific non-fungible token.'}),
+                        'doc': 'The address granted/denied proxy authority to manipulate non-fungible tokens.'}),
                     ('token', ('crypto:smart:token', {}), {
                         'doc': 'The specific token being granted access to.'}),
+                    ('alltokens', ('bool', {}), {
```

Comment:
This makes the `proxytokens` redundant ( and creates an odd state matrix for the `:token` property ) no?